app/test/vesting_test.go

```diff
@@ -0,0 +1,936 @@
+
+	// find and test a vesting account that has some vesting (locked) and
+	// some vested (unlocked) balance
+	for {
+		name := s.unusedAccount(PeriodicVestingAccountType)
+		address := getAddress(name, s.cctx.Keyring).String()
+
+		resAccBytes, err := GetRawAccountInfo(s.cctx.GRPCClient, address)
+		assert.NoError(s.T(), err)
+
+		var vAcc vestingtypes.PeriodicVestingAccount
+		err = vAcc.Unmarshal(resAccBytes)
+		assert.NoError(s.T(), err)
+
+		firstPeriodTime := vAcc.StartTime + vAcc.VestingPeriods[0].GetLength()
+		if firstPeriodTime < tmtime.Now().Unix() {
+			continue
+		}
+
+		// an account that has its first period passed by just a little
+		// and so only a part of allocation is unlocked
+		if firstPeriodTime >= tmtime.Now().Unix() {
+			waitTime := firstPeriodTime - tmtime.Now().Unix() + 1
+			s.T().Logf("waiting %d seconds...", waitTime)
+			time.Sleep(time.Duration(waitTime) * time.Second)
+		}
+		assert.False(s.T(), vAcc.GetVestedCoins(tmtime.Now()).IsZero(), "unlocked balance amount must not be zero")
+
+		s.testDelegatingVestingAmount(name)
+		return
+	}
+}
+
+func (s *VestingModuleTestSuite) TestGenesisPeriodicVestingAccountsClaimDelegationRewards() {
+	assert.NoError(s.T(), s.cctx.WaitForNextBlock())
+
+	// find and test a vesting account that has some vesting (locked) balance
+	// 	to be on the safe side we select one that starts unlocking in at least 20 seconds
+	for {
+		name := s.unusedAccount(PeriodicVestingAccountType)
+		address := getAddress(name, s.cctx.Keyring).String()
+
+		resAccBytes, err := GetRawAccountInfo(s.cctx.GRPCClient, address)
+		assert.NoError(s.T(), err)
+
+		var vAcc vestingtypes.PeriodicVestingAccount
+		err = vAcc.Unmarshal(resAccBytes)
+		assert.NoError(s.T(), err)
+
+		if vAcc.StartTime > tmtime.Now().Unix()+20 {
+			s.testDelegatingVestingAmount(name)
+			s.testClaimDelegationReward(name)
+			return
+		}
+	}
+}
+
+func (s *VestingModuleTestSuite) TestGenesisContinuousVestingAccountsSpendableBalance() {
+	assert.NoError(s.T(), s.cctx.WaitForNextBlock())
+
+	initCoinsForGasFee := sdk.NewCoin(app.BondDenom, sdk.NewInt(initBalanceForGasFee))
+
+	for {
+		name := s.unusedAccount(ContinuousVestingAccountType)
+		address := getAddress(name, s.cctx.Keyring).String()
+
+		resAccBytes, err := GetRawAccountInfo(s.cctx.GRPCClient, address)
+		assert.NoError(s.T(), err)
+
+		var vAcc vestingtypes.ContinuousVestingAccount
+		err = vAcc.Unmarshal(resAccBytes)
+		assert.NoError(s.T(), err)
+
+		balances, err := GetAccountSpendableBalance(s.cctx.GRPCClient, address)
+		assert.NoError(s.T(), err)
+
+		vestedCoins := vAcc.GetVestedCoins(tmtime.Now())
+		maxExpectedSpendableBalCoins := vestedCoins.Add(initCoinsForGasFee)
+		assert.LessOrEqual(s.T(),
+			balances.AmountOf(app.BondDenom).Int64(),
+			maxExpectedSpendableBalCoins.AmountOf(app.BondDenom).Int64())
+
+		// Stop testing once we hit an account with no spendable balance
+		if vestedCoins.IsZero() {
+			break
+		}
```

Comment:
to address that, the vesting accounts are generated using `startTime := tmtime.Now()` and it will increase by 5 seconds for each account. 
Since it takes a couple of seconds to start the network, there will always be some account with spendable balance.